packages/web/app/src/lib/preflight-sandbox/preflight-worker-embed.ts

```diff
@@ -129,6 +130,8 @@ function handleEvent(data: IFrameEvents.Incoming.EventData) {
           terminate();
           return;
         }
+
+        Kit.neverCase(ev.data);
```

Comment:
This is an exhaustiveness check. It helps improve the static type safety of our code by ensuring we don't accidentally forget to handle cases.